app/assets/stylesheets/application.scss

```diff
@@ -10,31 +11,82 @@ $font-stack: "PT Mono";
 // $primary-brand-color
 // $secondary-brand-color
 
+h1, h2, h3, h4, h5 {
+  font-family: 'Lora', serif;
+}
+
 $primary-background-color: $pastel-orange; //  pastel-orange
 $secondary-background-color: $light-green; // light-green
 // in overrides
-// $primary-color: #F35F55; //  pastel-red
-// $secondary-color: #335650; //dark-green
+$primary-color: #F35F55; //  pastel-red
+$secondary-color: #335650; //dark-green
 
 // Styles
 body {
-
+  background-color: #eaeee8;
   font-family: $font-stack;
   font-weight: normal;
 }
 
+a:hover, a:focus {
+    color: #FFFFFF;
+  }
+
+.park-index-list-item {
+  background-color: #d5edcd;
+  list-style-type: none;
+  padding: 20px 30px;
+  margin-bottom: 5px;
+}
+
+.index-details {
+    line-height: 2;
+    font-size: 100%;
+    color: #636363;
+    float: left;
+  }
+
+.index-rating p {
+  line-height: .6;
+  font-size: 300%;
+  color: #636363;
+  float: center;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+ }
+
+.park-info p {
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.park-icons img {
+  opacity: .6;
+  position: relative;
+  top: 50%;
+  transform: translateY(-50%);
+  margin-right: 10px;
+}
+
+.new {
+  color: #FFFFFF;
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`